cmd/utils/dev.go

```diff
@@ -236,9 +236,10 @@ const (
 func AskYesNo(q string, d YesNoDefault) (bool, error) {
 	var answer string
 	for {
-		oktetoLog.Question(fmt.Sprintf("%s %s: ", q, d))
-		_, err := fmt.Scanln(&answer)
-		if err != nil && err.Error() != "unexpected newline" {
+		if err := oktetoLog.Question(fmt.Sprintf("%s %s: ", q, d)); err != nil {
```

Comment:
Same with this one. If we can't throw a question we should return the proper error